pkg/cmd/pkg/package_install.go

```diff
@@ -1,26 +1,162 @@
 package pkg
 
 import (
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/dcos/client-go/dcos"
 	"github.com/dcos/dcos-cli/api"
+	"github.com/dcos/dcos-cli/pkg/plugin"
+	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
+
+	"github.com/dcos/dcos-core-cli/pkg/cosmos"
+	"github.com/dcos/dcos-core-cli/pkg/pluginutil"
 )
 
 func newCmdPackageInstall(ctx api.Context) *cobra.Command {
-	var appOnly, cliOnly, global, yes bool
+	var appOnly, cliOnly, yes bool
 	var appID, optionsPath, version string
 
 	cmd := &cobra.Command{
 		Use:   "install <package-name>",
 		Short: "Install a package",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return invokePythonCLI(ctx)
+			packageName := args[0]
+
+			// Install both if neither flag is specified
```

Comment:
I still don't get this, please bear with me. This check is true when either `cliOnly` and `appOnly` are true or both are false. This comment suggests it's only when both are false. I'd like to understand the reasoning behind this if.

Reply:
Passing `--app` means: `Only install the package on the cluster side but I don't want the CLI subcommand locally`.

Passing `--cli` means: `Skip the package installation and only install the CLI subcommand locally`

Passing both `--cli` and `--app` is not supported currently and returns with an error, these options are mutually exclusive, so having both `cliOnly` and `appOnly` should not happen, even though it is not enforced in this PR.

Reply:
>  both --cli and --app is not supported currently and returns with an error

I can't find the code that enforces this behaviour.

Reply:
By "currently", I meant with the Python command, this is not enforced anymore with this PR.

Reply:
Fixed